### PR TITLE
Fix logging setup when running in remote

### PR DESCRIPTION
**Pull request by nohwnd**
## Description
In remote scenario we can be passed path surrounded with quotes from parameters, and we should ignore those extra quotes. We should also create the full path for log, because in local run vstest.console handles that for us, and so testhost always has the full path, but in remote run the folder is not there and setting up logs fails, and nothing is logged

### CODE REVIEW

File: src/Microsoft.TestPlatform.CoreUtilities/Tracing/EqtTrace.cs [hunk 1]
```diff
@@ -161,6 +161,9 @@ public static bool InitializeVerboseTrace(string customLogFile)
     /// <returns>Trace initialized flag.</returns>
     public static bool InitializeTrace(string customLogFile, PlatformTraceLevel traceLevel)
     {
+        // Remove extra quotes if we get them passed on the parameter,
+        // System.IO.File does not ingore them when checking the file existence.
```
Comment (Evangelink) [inline src/Microsoft.TestPlatform.CoreUtilities/Tracing/EqtTrace.cs:165]:
```suggestion
        // System.IO.File does not ignore them when checking the file existence.
```

File: src/Microsoft.TestPlatform.PlatformAbstractions/common/Tracing/PlatformEqtTrace.cs [hunk 2]
```diff
@@ -369,6 +369,12 @@ private static bool EnsureTraceIsInitialized()
                     runnerLogFileName = LogFile;
                 }
 
+                var runnlerLogFileInfo = new FileInfo(runnerLogFileName);
+                if (!Directory.Exists(runnlerLogFileInfo.DirectoryName))
+                {
+                    Directory.CreateDirectory(runnlerLogFileInfo.DirectoryName);
+                }
```
Comment (Evangelink) [inline src/Microsoft.TestPlatform.PlatformAbstractions/common/Tracing/PlatformEqtTrace.cs:376]:
```suggestion
                var runnerLogFileInfo = new FileInfo(runnerLogFileName);
                if (!Directory.Exists(runnerLogFileInfo.DirectoryName))
                {
                    Directory.CreateDirectory(runnerLogFileInfo.DirectoryName);
                }
```

**Review by Evangelink — APPROVED** [review]

File: src/Microsoft.TestPlatform.CoreUtilities/Tracing/EqtTrace.cs (hunk 1 above)
Comment (nohwnd) [reply src/Microsoft.TestPlatform.CoreUtilities/Tracing/EqtTrace.cs:165]: fixed

File: src/Microsoft.TestPlatform.PlatformAbstractions/common/Tracing/PlatformEqtTrace.cs (hunk 2 above)
Comment (nohwnd) [reply src/Microsoft.TestPlatform.PlatformAbstractions/common/Tracing/PlatformEqtTrace.cs:376]: fixed

File: src/Microsoft.TestPlatform.PlatformAbstractions/common/Tracing/PlatformEqtTrace.cs [hunk 3]
```diff
@@ -367,6 +367,12 @@ private static bool EnsureTraceIsInitialized()
                     runnerLogFileName = LogFile;
                 }
 
+                var runnerLogFileInfo = new FileInfo(runnerLogFileName);
+                if (!Directory.Exists(runnerLogFileInfo.DirectoryName))
```
Comment (Evangelink) [inline src/Microsoft.TestPlatform.PlatformAbstractions/common/Tracing/PlatformEqtTrace.cs:371]: Nit: You could do `runnerLogFileInfo.Directory.Exists` instead.

**Review by Evangelink — APPROVED** [review]
Approved with 1 nit.

**Pull request merged**